src/handlers/testing/exec/components/progress/index.tsx

```diff
@@ -296,24 +312,7 @@ const App = (props: AppProps) => {
         width="100%"
         flexDirection="column"
       >
-        {autoblocksCIBuildHtmlUrl && (
-          // Hardcode the width so that the URL doesn't wrap.
-          <Box width={`View results at ${autoblocksCIBuildHtmlUrl}`.length}>
-            <Text color="gray">View results at</Text>
-            <Space />
-            <Text color="cyan" dimColor={true}>
-              {autoblocksCIBuildHtmlUrl}
-            </Text>
-          </Box>
-        )}
-        <Box
-          paddingX={1}
-          flexDirection="column"
-          borderStyle="round"
-          borderColor="gray"
```

Comment:
i got rid of the border b/c it gets screwed up too easily both in ci and locally if you resize